### PR TITLE
add project URL to csproj file

### PR DESCRIPTION
**Pull request by swharden**
This will help connect the NuGet package with this GitHub repository. The project URL is displayed on the package page on the NuGet website, and also on the NuGet package manager integrated with Visual Studio.

### CODE REVIEW

File: src/Namotion.Reflection/Namotion.Reflection.csproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <PackageIconUrl>https://raw.githubusercontent.com/RicoSuter/Namotion.Reflection/master/assets/NuGetIcon.png</PackageIconUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Description>.NET library with advanced reflection APIs like XML documentation reading, Null Reference Types (C# 8) reflection and string based type checks.</Description>
+    <projectUrl>https://github.com/RicoSuter/Namotion.Reflection</projectUrl>
```
Comment (jeremyVignelles) [inline src/Namotion.Reflection/Namotion.Reflection.csproj:14]: Aren't the properies case sensitive?

**Review by jeremyVignelles — COMMENTED** [review]
Thanks for the PR! 🙂

File: src/Namotion.Reflection/Namotion.Reflection.csproj (hunk 1 above)
Comment (swharden) [reply src/Namotion.Reflection/Namotion.Reflection.csproj:14]: I made this PR in my browser, and I think there was a copy/paste error.

I'll modify it to be identical to what I'm currently using
https://github.com/ScottPlot/ScottPlot/blob/486f2ae38b9a4edb8eeeb1e8bc92c5e1a425ffca/src/ScottPlot/ScottPlot.csproj#L13

And that shows up well on NuGet under the Info section as "project site"
https://www.nuget.org/packages/ScottPlot/

**Comment by swharden** [comment]
I updated this PR to use `PackageProjectUrl` similar to how NJsonSchema does it: https://github.com/RicoSuter/NJsonSchema/blob/fd752522bdb46eb3bad5b45bae86b4789744d709/src/NJsonSchema/NJsonSchema.csproj#L8

I confirmed that `projectUrl` is the field name for nuspec manifest files (not csproj files)

**Comment by RicoSuter** [comment]
Thx for the PR. Can you also add this to the cecil package?

**Comment by swharden** [comment]
Hi @RicoSuter, I made the change you suggested.

Notice I also switched Cecil's license URL to a license expression to bring it into consistentency with the Reflection package. I think the expression is preferred where possible.

https://github.com/RicoSuter/Namotion.Reflection/blob/4279901776ed8cbcb9f2338cefd1ab5465948863/src/Namotion.Reflection/Namotion.Reflection.csproj#L12

**Comment by RicoSuter** [comment]
Super, thanks for the PR!

**Pull request merged**